random.js

```diff
@@ -1,9 +1,96 @@
-var jugadores = ["ignacio","victor","nil","camaleonte","diodoro","rober","lozano","pepe","adrian","ramon","jimeno","carlos"];
-var dado = Math.floor(Math.random() * jugadores.length);
-var mensaje = "<h1>"+jugadores[dado]+"</h1>";
```

Comment:
It's a good practice to only use english in anything related to programming, since everything is in english (even the languages themselves). This way also allow anyone to understand what you've written or commented, you never know who'll end up using your code